contracts/src/systems/actions.cairo

```diff
@@ -90,7 +91,6 @@ mod actions {
             let mut mancala_game: MancalaGame = get!(world, game_id, (MancalaGame));
 
             // need to create validation for this status
-            assert!(mancala_game.status != GameStatus::Finished, "Game is already finished");
```

Comment:
i removed this because the check about Gamestatus is done in validate_Move anyway